### ISSUE TITLE
production: "no global services for lease"

### ISSUE THREAD

**Issue by boz** [ovrclk/akash#1228]
Seems to happen sporadically.  From user `tombeyond` on discord:

> FYI I keep getting no global services for lease, for both of the providers that tend to bid. Resolved itself at least once by changing > provider but hasn't worked the last few times. Don't think I'm doing anything stupid otherwise I'd post on #deployments :joy:
> `akash provider lease-status --from deploy --dseq 647920 --oseq 1 --gseq 1 --provider akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal`

**Comment by troian** [comment]
related https://github.com/ovrclk/akash/pull/1216 ?

**Comment by tombeynon** [comment]
Thanks for raising this @boz.

I think this is part of a wider problem with closed deployments continuing to run. In this instance, I'm trying to deploy a simple container with the hostname akashhw.tombeynon.co.uk. This was previously deployed on Akash and I closed the deployment about a week ago. However the site is still running at the original Akash hostname: 8lim5k05ft9bhdm12vpvrghcr8.ingress.ewr1p0.mainnet.akashian.io

When I try to re-deploy (having closed the old deployment), I get this 'no global services for lease' when querying the lease-status. I believe what's happening is the deployment hasn't closed properly, is still using the vanity domain I'm specifying, therefore it's not available for a new deployment. I can't close the old deployment as it's already closed from my point of view. 

Another example is the following Akash node running on Akash. I closed this days ago and it's still up and syncing - http://176ddd8tk9827439rjnl09es0g.ingress.ewr1p0.mainnet.akashian.io:32006/status.

Deploy.yml included below, just trying to deploy this will throw the same error: 

```
---
version: "2.0"

services:
  web:
    image: tombeynon/akash-hello-world:release-v0.1.1
    expose:
      - port: 80
        as: 80
        accept:
          - akashhw.tombeynon.co.uk
        to:
          - global: true

profiles:
  compute:
    web:
      resources:
        cpu:
          units: 0.5
        memory:
          size: 512Mi
        storage:
          size: 512Mi
  placement:
    dcloud:
      attributes:
        host: akash
      signedBy:
        anyOf:
          - "akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63"
      pricing:
        web:
          denom: uakt
          amount: 1

deployment:
  web:
    dcloud:
      profile: web
      count: 1
```

**Comment by boz** [comment]
possible dupe of #1227

**Comment by troian** [comment]
will be fixes as part of #1227
